pyvista/utilities/geometric_objects.py

```diff
@@ -1288,9 +1288,15 @@ def Circle(radius=0.5, resolution=100):
     >>> radius = 0.5
     >>> circle = pyvista.Circle(radius)
     >>> circle.plot(show_edges=True, line_width=5)
+
+    Notes
+    --------
+    prior to version 0.38,
+        this method had incorrect results, which are now fixed.
+    See pull request 3710 for details.
```

Comment:
1. I would use the `.. versionchanged:` directive for this, assuming that's visible enough. We can check existing examples.
2. Instead of linking to the PR I think we should explain the change in a few words. Most end users don't want to see where the sausage is made :)

Reply:
Fixed